client/src/components/CaptureMatching/CandidateImages.js

```diff
@@ -0,0 +1,157 @@
+import React, {useState, useRef} from 'react'
+
+import { Typography, Box, Button, GridList, Grid} from '@material-ui/core';
+import {makeStyles} from '@material-ui/core/styles'
+import ZoomOutMapIcon from '@material-ui/icons/ZoomOutMap';
+import CheckIcon from '@material-ui/icons/Check';
+import ClearIcon from '@material-ui/icons/Clear';
+
+
+
+const useStyles = makeStyles({
+
+    containerBox: {
+        margin: '25px 20px 10px 20px',
+        paddingTop: '20px',
+        paddingBottom: '10px',
+       background: '#fff',
+       borderRadius: '4px',
+
+
+    },
+
+    headerBox: {
+        display: 'flex',
+        flexDirection: 'spaceBetween'
+    },
+
+
+    imgContainer: {
+        width: '350px',
+        height: 'auto',
+        padding: '5px',
+         objectFit: 'cover',
+        paddingBottom: '10px',
+        overFlow: 'hidde',
+    },
+    gridList: {
+
+         padding: '10px',
+         display: 'flex',
+         flexDirection: 'row',
+         overflowX: 'auto'
+  },
+
+        imageScroll: {
+        // height: '100vh',
+        height: '830px',
```

Comment:
I can help out with getting this to fill the viewport. We do something similar in the _Verify_ page (`TreeImageScrubber.js`).